extensions/spark/kyuubi-spark-authz/src/main/resources/table_command_spec.json

```diff
@@ -2033,7 +2033,17 @@
   } ],
   "opType" : "QUERY",
   "queryDescs" : [ ],
-  "uriDescs" : [ ]
+  "uriDescs" : [ {
+    "fieldName" : "clone",
+    "fieldExtractor" : "HudiCallProcedureInputUriExtractor",
+    "isInput" : true,
+    "comment" : "Hudi"
```

Comment:
I think we don't need to add this here, better in the whole hudi node.

Reply:
thanks @AngersZhuuuu for the suggestion.

Maybe I misunderstood your meaning, do you mean the 'comment' attribute that can be removed?

Reply:
> thanks @AngersZhuuuu for the suggestion.
> 
> Maybe I misunderstood your meaning, do you mean the 'comment' attribute that can be removed?

Yea, if you want to add comment to define query type, we can add it in top level 
``` {
  "classname" : "org.apache.spark.sql.hudi.command.CallProcedureHoodieCommand",
  "comment": "Hudi"
````

Reply:
As far as I know, the comment field is only used for [error output](https://github.com/apache/kyuubi/blob/ac388d191ddb6a910a5c74575d3ba6aa5a31e062/extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/serde/Descriptor.scala#L64)(maybe I missed something 👽), So I don't understand what the purpose of deleting this field is?

BTW, the top level CommandSpec has no comment field.